decidim-forms/app/cells/decidim/forms/step_navigation/show.erb

```diff
@@ -1,35 +1,23 @@
-<div class="button--double form-general-submit answer-questionnaire__submit">
-  <% if first_step? %>
-    <a></a>
-  <% else %>
-    <%= link_to(
-      icon("caret-left", class: "icon--small", role: "img", "aria-hidden": true) + " " + t("decidim.forms.step_navigation.show.back"),
-      "#",
-      class: "hollow secondary",
-      data: {
-        toggle: [previous_step_dom_id, current_step_dom_id].join(" ")
-      }
-    ) %>
+<div class="form-general-submit answer-questionnaire__submit">
+  <% if !first_step? %>
+    <%= link_to("#", class: "arrow-link text-lg text-secondary", data: { toggle: [previous_step_dom_id, current_step_dom_id].join(" ") }) do %>
```

Comment:
Use a `<button>` when the element is not a link i.e. does not link to some other URL.